packages/core-mobile/app/screens/send/utils/evm/send.ts

```diff
@@ -93,18 +92,10 @@ const getTransactionRequest = ({
         toAddress,
         amount
       })
-      const gasLimit = await getGasLimit({
-        fromAddress,
-        provider,
-        token,
-        toAddress,
-        amount: amount ?? 0n
-      })
 
       return {
         ...txRequest,
-        chainId,
-        gasLimit
+        chainId
```

Comment:
extension doesn't really pass gas limit to vm module so we should do the same here. vm module will calculate gas limit when it is not provided so we should be good.